src/component/content.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+
+class Content extends React.Component{
```

Comment:
```suggestion
class Content extends React.Component {
```
no space looks weird. code style guidelines are helpful. prettier or eslint look into